buildspec.yml

```diff
@@ -0,0 +1,20 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      java: corretto11
+  pre_build:
+    commands:
+      - echo Nothing to do in the pre_build phase...
+  build:
+    commands:
+      - echo Build started on `date`
+      - mvn clean install
+      - echo Running unit tests
+      - mvn clean test
```

Comment:
This isn't necessary since Travis is already running the unit tests

Reply:
Removed unit tests